src/__tests__/a-reduce.test.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env jest */
+import a, { isAsync } from '../macros/sync-to-async.macro'
```

Comment:
I would prefer using a more descriptive name instead of "a". Possibly containing the word "macro"
"syncToAsyncMacro" ?